webapp/src/auth.ts

```diff
@@ -66,43 +67,42 @@ class AuthClient extends pxt.auth.AuthClient {
         return Promise.resolve();
     }
     protected async onProfileDeleted(userId: string): Promise<void> {
-        // Convert cloud-saved projects to local projects.
-        await cloud.convertCloudToLocal(userId);
+        try {
+            // Convert cloud-saved projects to local projects.
+            await cloud.convertCloudToLocal(userId);
```

Comment:
Do you want a tick event for the successful case as well?

Reply:
We were more interested in just knowing whether if this ever failed, but having a success tick would make it easier to get the rate of failure. We have other ticks we can use in its place - notably `auth.profile.deleted` - so I think we're covered if that question comes up.